### ISSUE TITLE
Ctrl+C doesn't work on macOS

### ISSUE THREAD

**Issue by vors** [PowerShell/PSReadLine#615]
Environment data
----------------

```
PS version: 6.0.0
PSReadline version: 2.0.0-beta1
os: Darwin [redacted] 16.7.0 Darwin Kernel Version 16.7.0: Mon Nov 13 21:56:25 PST 2017; root:xnu-3789.72.11~1/RELEASE_X86_64 x86_64
```

Steps to reproduce or exception report
--------------------------------------

Install 2.0.0-beta1 pre-release with `Install-Module -AllowPrerelease -AllowClobber -Scope CurrentUser PSReadLine -Force`
Open a new powershell and try to use `Ctrl+C` to cancel the line.
It works on 1.2, but on 2.0.0 it does nothing.

**Comment by barcexwiki** [comment]
Same thing on Linux:
```
PS > Get-Module

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   3.1.0.0    Microsoft.PowerShell.Management     {Add-Content, Clea...
Manifest   3.1.0.0    Microsoft.PowerShell.Utility        {Add-Member, Add-T...
Script     2.0.0      PSReadLine                          {Get-PSReadLineKey...


PS > $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.0.0
PSEdition                      Core
GitCommitId                    v6.0.0
OS                             Linux 4.10.0-28-generic #32~16.04.2-Ubuntu SM...
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

But not on Windows

```
PS C:\temp\PowerShell\debug> Get-Module

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   3.1.0.0    Microsoft.PowerShell.Management     {Add-Content, Clear-Content, Clear-Item, Clear-ItemProperty...}
Manifest   3.1.0.0    Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object...}
Script     2.0.0      PSReadLine                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PSReadLineKeyHandler, Set-PSReadLineKeyHandler...}

PS C:\temp\PowerShell\debug> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.0.0
PSEdition                      Core
GitCommitId                    v6.0.0
OS                             Microsoft Windows 10.0.16299
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```

**Comment by lzybkr** [comment]
This was fixed with 09b693014e3958219a491ffd23fa0bb2908d441a which is not in beta1.

With PowerShell 6.0 GA, more folks are trying beta1, so hopefully I can get a few more important bugs fixed before I publish beta2.

**Comment by jbvmio** [comment]
Any idea when beta2 will be released or if we can fix this temporarily in the meantime? This is quite an important feature currently missing.

Disregard the above, I just found another thread here stating you have builds on AppVeyor and found the latest. Installed and working now.

https://ci.appveyor.com/project/lzybkr/PSReadLine/build/artifacts

Thanks.
